app/controllers/favorites_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
 
   def destroy
     pet = Pet.find(params[:pet_id])
-    favorites.destroy_pet(pet)
+    favorites.destroy_pet(pet.id)
     flash[:notice] = "You have removed #{pet.name} from your favorites"
```

Comment:
Oh!  Here's the flash notice from before!  Where did you get the destroy_pet method from?

Reply:
Had to create it like the add pet method (but instead of shoveling we just delete)